cocotb/share/lib/simulator/simulatormodule.c

```diff
@@ -495,9 +603,11 @@ static PyObject *register_value_change_callback(PyObject *self, PyObject *args)
     }
 
     PyObject *pSihHdl = PyTuple_GetItem(args, 0);
```

Comment:
```suggestion
    PyObject *pSigHdl = PyTuple_GetItem(args, 0);
```
(and usage below obviously too)

Reply:
Good catch.